source/app.d

```diff
@@ -85,25 +88,54 @@ struct AppConfig
 	string ghuser, ghpassword,
 		   glurl, glauth,
 		   bbuser, bbpassword;
+
+	bool enforceCertificateTrust = false;
+
+	string mailServer;
+	ushort mailServerPort;
+	SMTPConnectionType mailConnectionType;
+	string mailClientName;
+	string mailUser;
+	string mailPassword;
+
+
 	void init()
 	{
 		import dub.internal.utils : jsonFromFile;
 		auto regsettingsjson = jsonFromFile(NativePath("settings.json"), true);
+		// TODO: use UDAs instead
```

Comment:
JSON with environment fallback might also be a common use-case we would want to support in vibe.d